lib/jblond/Diff.php

```diff
@@ -50,8 +50,9 @@ class Diff
     private $groupedCodes;
 
     /**
-     * @var array   Associative array containing the default options available for the diff class and their default
-     *              value.
+     * @var array<string> Associative array containing the default options available
```

Comment:
It's a new syntax I never use, but shouldn't it be `array<string, string>` in this case?

Reply:
That is the syntax for an array of strings. other example would be array<string|array>
__

Reply:
I've got my info from https://suckup.de/2020/02/modern-phpdoc-annotations/
It's all fine with me as long as it's compatible with PhpStorm 😇